source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/files/styles.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../../../styles/_base/variables.scss';
+
+.product-artifact {
+  border: 1px solid $lighter-gray;
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+  color: $gray;
+
+  .header {
+    background-color: $ultra-ligthest-gray;
+    padding: 17px 22px;
+    font-size: 16px;
+    font-weight: bold;
+    font-family: Montserrat;
```

Comment:
it is

Reply:
so why specify all this extra stuff? makes maintenance harder.

Reply:
To start with we don't inherit font-family to all elements so if I don't specify it won't be montserrat. Font size can be remove and font size same as font-family

and eventually this is even harder to mantain

<div className='header flex align-items-center p-t-sm p-b-sm p-l-md p-r-md'> //Imagine adding classes for a lot of stuff here.

Reply:
if montserrat isn't the default font, then we should make it the default. otherwise we're going to have a toooooooooooooooooooooooooooon of repetition